### DISCUSSION TITLE
Gavrilo flýr Viborg - 2023/8/5

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#36] [vefsida-ummæli]
# Gavrilo flýr Viborg - 2023/8/5

Gavrilo tókst að flýja Viborg

https://www.biskupsreglan.com/verkefni/gavrilo-i-viborg

**Comment by MatthiasGudni** [comment]
Þetta eru gleði féttir, kannski tekst Gavrilo að finna mig í oktober

**Reply by MatthiasGudni** [reply]
Það er fullt af fólki að tala skringilega í kringum mig

**Reply by TJ478** [reply]
Þú verður að fara varlega
